index.js

```diff
@@ -6,12 +6,16 @@
  * @param  {Boolean} options.hasHeader - this will determine if the table data has a header or not, by default this is true
  * @param  {String}  options.columnSeparator - the default separators is ' | '
  * @param  {String}  options.headerSeparator - the default header separators is '=', this value should only be one character
- * @return {String} - a string that represents the ascii table of the data provided
+ * @return {Object} - { ascii - a string that represents the ascii table of the data provided,
+ *                      html - a provided html version of table,
+ *                      table - html <table> table,
+ *                      markdown - a markdown table }
  */
 module.exports = function turtler(data, options={}) {
   if(!Array.isArray(data)) throw new Error('data should be an array of arrays');
 
-  let table = '';
+  let asciiTable = '';
```

Comment:
Yeah I am sure that this is a good addition but should be retrieved using a getter method.

for example

```
get ascii () {
 return this.toString();
}
get markdown() {
 return // code to make it a marko
}
```